frontend/src/components/MapView/MapChat/index.js

```diff
@@ -108,66 +105,69 @@ class MapChat extends Component {
   }
 
   render = () => {
-    const rightOffset = this.state.open ? '0' : '-300px'
+    const rightOffset = this.props.chatOpen ? '0' : '-300px'
     const { conversationLive, isParticipating, participants, messages, inviteACall, inviteToJoin } = this.props
     const { videosShowing, cursorsShowing, alertSound, unreadMessages } = this.state
     return (
-      <div className="chat-box"
-        style={{ right: rightOffset }}
-      >
-        <div className="junto-header">
-          PARTICIPANTS
-          <div onClick={this.toggleVideosShowing} className={`video-toggle ${videosShowing ? '' : 'active'}`} />
-          <div onClick={this.toggleCursorsShowing} className={`cursor-toggle ${cursorsShowing ? '' : 'active'}`} />
+      <div id="chat-box-wrapper">
```

Comment:
at the moment, it still serves a purpose from a page layout/styling perspective, so I'm going to leave it in, but it'll become a lot easier to remove it after this